src/base/bittorrent/session.cpp

```diff
@@ -3044,13 +3049,18 @@ void Session::startUpTorrents()
         QByteArray data;
     } TorrentResumeData;
 
-    auto startupTorrent = [this, logger, resumeDataDir](const TorrentResumeData &params)
+    int addCount = 0;
+    const auto startupTorrent = [this, &logger, &resumeDataDir, &addCount](const TorrentResumeData &params)
```

Comment:
As to me, there are two reasons why to avoid such kind of a code:
1. Expressive means of C++ include almost every language detail. C++ programmers usually enjoy this expressiveness to every single bit of it. Thus, when a bit if this expressiveness is used without a reason, a regular C++ programmer does not accept that.
2. A reference is applied to a function parameter when one wants to change its value from inside of the function. A reader would expect this and when the finds out that this is not the case, he, likely, will re-read this fragment a couple of times and a feeling that something is not right here will stay with him.